applications/StructuralMechanicsApplication/tests/rigid_test/sphere_failing_parameters.json

```diff
@@ -79,7 +79,7 @@
                     "model_part_name" : "Structure.SelfWeight3D_Self_weight_Auto1",
                     "variable_name"   : "VELOCITY_Z",
                     "interval"        : [0.0, "End"],
-                    "value"           : "-9.81*(t + 0.05)",
+                    "value"           : "-9.81*t",
```

Comment:
in the scope of this PR, this makes sense to me

Reply:
Indeed